src/Microsoft.ML.FastTree/Dataset/DenseIntArray.cs

```diff
@@ -70,13 +71,14 @@ public override IntArray[] Split(int[][] assignment)
         }
 
 #if USE_FASTTREENATIVE
-        [DllImport("FastTreeNative", CallingConvention = CallingConvention.StdCall)]
+        internal const string NativePath = "FastTreeNative";
+        [DllImport(NativePath), SuppressUnmanagedCodeSecurity]
```

Comment:
@jkotas - is the `SuppressUnmanagedCodeSecurity` attribute necessary/useful on either .NET Core or .NET Framework 4.6.1+? I'm wondering if we should just get rid of it in our code.

Reply:
It is useless for .NET Core. It is needed for good perf on .NET Framework, unfortunately.